plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -0,0 +1,196 @@
+  | 'documentation'
+  | 'api'
+  | 'tool';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    card: {
+      border: `1px solid ${theme.palette.divider}`,
+      boxShadow: theme.shadows[2],
+      borderRadius: '4px',
+      padding: theme.spacing(2),
+      color: '#fff',
+      transition: `${theme.transitions.duration.standard}ms`,
+      '&:hover': {
+        boxShadow: theme.shadows[4],
+      },
+    },
+    bold: {
+      fontWeight: theme.typography.fontWeightBold,
+    },
+    service: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.service.colors})`,
+    },
+    website: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.website.colors})`,
+    },
+    library: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.library.colors})`,
+    },
+    documentation: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.documentation.colors})`,
+    },
+    api: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, #005B4B, #005B4B)`,
+    },
+    tool: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.tool.colors})`,
+    },
+  }),
+);
+
+const countEntitiesBy = (
+  entities: Array<Entity>,
+  kind: EntitiesKinds,
+  type?: EntitiesTypes,
+) =>
+  entities.filter(
+    e => e.kind === kind && (type ? e?.spec?.type === type : true),
+  ).length;
+
+const EntityCountTile = ({
+  counter,
+  className,
+  name,
+}: {
+  counter: number;
+  className: EntitiesTypes;
+  name: string;
+}) => {
+  const classes = useStyles();
+  return (
+    <Box
+      className={`${classes.card} ${classes[className]}`}
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+    >
+      <Typography className={classes.bold} variant="h6">
+        {counter}
+      </Typography>
+      <Typography className={classes.bold} variant="h6">
+        {name}
+      </Typography>
+    </Box>
+  );
+};
+
+export const OwnershipCard = ({
+  entity,
+  variant,
+}: {
+  entity: Entity;
+  variant: string;
+}) => {
+  const {
+    metadata: { name: groupName },
+  } = entity;
+  const catalogApi = useApi(catalogApiRef);
+  const {
+    loading,
+    error,
+    value: componentsWithCounters,
+  } = useAsync(async () => {
+    const entitiesList = await catalogApi.getEntities();
+    const ownedEntitiesList = entitiesList.items.filter(component =>
+      component?.relations?.some(
+        r => r.type === RELATION_OWNED_BY && r.target.name === groupName,
+      ),
+    ) as Array<Entity>;
+
+    return [
+      {
+        counter: countEntitiesBy(ownedEntitiesList, 'Component', 'service'),
+        className: 'service',
+        name: 'Services',
+      },
+      {
+        counter: countEntitiesBy(
+          ownedEntitiesList,
+          'Component',
+          'documentation',
+        ),
+        className: 'documentation',
+        name: 'Documentations',
```

Comment:
```suggestion
        name: 'Documentation',
```

Minor note: would read better without the S here